sagemaker-python-sdk/tensorflow_script_mode_using_shell_commands/tensorflow_script_mode_using_shell_commands.ipynb

```diff
@@ -0,0 +1,303 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# TensorFlow Script Mode - Using Shell commands\n",
+    "\n",
+    "Starting by TensorFlow's framework version 1.11, you can shell commands as\n",
+    "your training entry point. Shell scripts are useful for many use cases including:\n",
+    "\n",
+    "- Invoke Python scripts with specific parameters\n",
+    "- Install requirements before training\n",
+    "- Configure framework dependencies\n",
+    "- Train using different computer languages\n",
+    "\n",
+    "For this example, you use [a Keras implementation of the Deep Dream algorithm](https://github.com/keras-team/keras/blob/2.2.4/examples/deep_dream.py). You can use the same technique for other scripts or repositories. For example, [TensorFlow Model Zoo](https://github.com/tensorflow/models) and [TensorFlow benchmark scripts](https://github.com/tensorflow/benchmarks/tree/master/scripts/tf_cnn_benchmarks)."
```

Comment:
change the first "you" to a "we". same comment as before about the "for example" clause